adafruit_emc2101/__init__.py

```diff
@@ -299,8 +325,11 @@ def manual_fan_speed(self):
         """The fan speed used while the LUT is being updated and is unavailable. The speed is
         given as the fan's PWM duty cycle represented as a float percentage.
         The value roughly approximates the percentage of the fan's maximum speed"""
-        raw_setting = self._fan_setting & EMC2101_Regs.MAX_LUT_SPEED
-        return (raw_setting / self._full_speed_lsb) * 100
+        raw_setting = self._fan_setting & emc2101_regs.MAX_LUT_SPEED
+        fs = self._full_speed_lsb
```

Comment:
'self._full_speed_lsb' is however not a simple variable (if I have understood properly) but code that ends up doing operations that take significant amount of time.

If the code is running on an MCU, that additional time isn't too desperate (something like 0.05ms) but when it's then pushed over a USB interface via the FT232H the round trip time turns into milliseconds. Because this is exactly what I'm doing I want to keep unnecessary reads to a minimum.

I'll rename fs to fan_speed.